src/utils/translation.ts

```diff
@@ -27,6 +28,7 @@ export const supportedLanguages = new Map<string, string>([
   ["fr", "français"],
   ["hi", "हिन्दी"],
   ["ru", "русский"],
+  ["fa", "پارسی"],
```

Comment:
We doesn't support rtl languages yet. Could you please move this language to the Beta section?

Reply:
Consider it done!